io/GDALWriter.cpp

```diff
@@ -321,6 +329,45 @@ void GDALWriter::doneFile()
         throwError(raster.errorMsg());
 
     getMetadata().addList("filename", m_filename);
+
+    std::vector<std::string> gdalitems = Utils::split(m_GDAL_metadata, ',');
+    for (auto& v: gdalitems)
+    {
+        const std::size_t pos = v.find_first_of("=");
+        if (pos != std::string::npos)
+        {
+            const std::string name = v.substr(0, pos);
+            const std::string value = pos != std::string::npos ? v.substr(pos + 1) : "";
+            raster.addMetadata(name, value);
+        }
+    }
+
+    getMetadata().addList(raster.getMetadata());
+}
+
+void GDALWriter::readyTable(PointTableRef table)
+{
+    // Add these as GDAL metadata
+    if(m_writePDALMetadata)
+    {
+        MetadataNode m = table.metadata();
+        std::string json = Utils::toJSON(m);
+        std::vector<uint8_t> metadata(json.begin(), json.end());
+        std::string b64 = Utils::base64_encode(metadata.data(), metadata.size());
```

Comment:
I don't understand why you'd be encoding a string to a string. Why not just save the string without encoding?

Reply:
> Why not just save the string without encoding?

Because if they give us JSON or any other weird thing we don't have to worry about escaping it upon output anywhere – it's just a blob of crap that the downstream user is responsible for.